Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
Traceback (most recent call last):
  File "_4_data_input_prep_ptyREX.py", line 9, in <module>
    import numpy as np
ImportError: No module named numpy
